.gitignore

```diff
@@ -26,6 +26,7 @@ platforms/
 plugins/
 plugins/android.json
 plugins/ios.json
+www/
```

Comment:
why was this added to the .gitignore? This folder needs to be added to the repo in order to start an app with the conference app without errors, e.g. `ionic start --v2 myApp conference`